tests/spec/w3c/linter-rules/required-sections-spec.js

```diff
@@ -11,7 +11,11 @@ import { noTrackStatus } from "../../../../src/w3c/headers.js";
 import { requiresSomeSectionStatus } from "../../../../src/w3c/linter-rules/required-sections.js";
 
 describe("w3c — required-sections", () => {
+  beforeAll(() => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL *= 2;
```

Comment:
I think the w3c server was being slow tonight... I don't think we actually need this.

Reply:
Yeah, let's not forget to remove this. Or, we should probably set it at config level as it's a "timeout".